export/build.gradle

```diff
@@ -52,10 +52,10 @@ repositories {
 }
 
 dependencies {
-    compile 'com.google.cloud:google-cloud-pubsub:1.89.0'
+    compile 'com.google.cloud:google-cloud-pubsub:1.98.0'
     compile 'com.google.guava:guava:26.0-jre'
     compile "com.beust:jcommander:1.72"
-    compile 'com.google.auth:google-auth-library-oauth2-http:0.9.0'
+    compile 'com.google.auth:google-auth-library-oauth2-http:0.18.0'
     compile "com.google.api-client:google-api-client:1.23.0"
```

Comment:
What about this line, should it also be updated?

Can you please add a bit more information about what's happening, and how we can prevent it / know that this issue won't happen again?

Reply:
```
Exception in thread "main" java.lang.NoSuchMethodError: 'com.google.protobuf.Internal$IntList com.google.protobuf.DescriptorProtos$FileDescriptorProto.emptyIntList()'
	at com.google.protobuf.DescriptorProtos$FileDescriptorProto.<init>(DescriptorProtos.java:1175)
	at com.google.protobuf.DescriptorProtos$FileDescriptorProto.<clinit>(DescriptorProtos.java:4696)
	at com.google.protobuf.Descriptors$FileDescriptor.internalBuildGeneratedFileFrom(Descriptors.java:339)
	at com.google.cloud.healthcare.deid.redactor.protos.DicomConfigProtos.<clinit>(DicomConfigProtos.java:2082)
	at com.google.cloud.healthcare.deid.redactor.protos.DicomConfigProtos$DicomConfig$Builder.getDescriptorForType(DicomConfigProtos.java:1437)
	at com.google.protobuf.MessageReflection$BuilderAdapter.getDescriptorForType(MessageReflection.java:374)
	at com.google.protobuf.TextFormat$Parser.mergeField(TextFormat.java:1501)
	at com.google.protobuf.TextFormat$Parser.mergeField(TextFormat.java:1484)
	at com.google.protobuf.TextFormat$Parser.merge(TextFormat.java:1468)
	at com.google.protobuf.TextFormat$Parser.merge(TextFormat.java:1387)
	at com.google.cloud.healthcare.deid.redactor.DicomRedactor.parseConfig(DicomRedactor.java:130)
	at com.google.cloud.healthcare.deid.redactor.DicomRedactor.<init>(DicomRedactor.java:84)
	at com.google.cloud.healthcare.imaging.dicomadapter.ImportAdapter.configureRedactor(ImportAdapter.java:145)
	at com.google.cloud.healthcare.imaging.dicomadapter.ImportAdapter.main(ImportAdapter.java:94)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at com.intellij.rt.execution.application.AppMainV2.main(AppMainV2.java:131)
```

Redactor lib is built with com.google.protobuf:protobuf-java:3.7.1 , but ends up running with 3.6.1 and finds one of few things that actually differ between versions. Remove/Keep lists have shorter and more simple execution paths within DicomRedactor.parseConfig, so they don't hit the problem.
Why: redactor lib is plugged in as is (I guess this is a good reason to avoid including jars like this in general), while 3.6.1 comes from com.google.cloud:google-cloud-monitoring:1.59.0 .

So the solution is to update dependencies which brought out of date protobuf to more modern versions. I updated versions in more cases than minimally required to fix the conflict once I noticed that I had some dependencies in triplicate (3 versions of same thing from different dependency chains).

Reply:
Managing these dependencies ourselves between the JAR and our gradle files seems problematic. 

I think the best solution would be to remove the dependency on the JAR file. Let's go back to building from the clone jitpack repo, since that seems like our only option here given how importing the JAR has a high risk of dependency conflict.

Reply:
Ok, returned to jitpack.